index.html

```diff
@@ -103,18 +106,19 @@ <h3 class="article__title">How can I help?</h3>
               </div>
             </div>
           </div>
-        </div>
+        </main>
 
-        <div class="sidebar">
+        <sidebar class="sidebar">
```

Comment:
`sidebar` is different from `aside`. 
In short, `aside` is a html tag but `sidebar` is just the name of an element in the website layout.
Please read more here: https://aastudio.fr/Sidebar-and-Aside-are-different.html

Reply:
Thank you Mahsa for your feedback. i checked the link you send me and i understood what i did wrong